src/Umbraco.Community.Contentment/DataEditors/ConfigurationEditor/configuration-editor.js

```diff
@@ -231,6 +231,17 @@ angular.module("umbraco").controller("Umbraco.Community.Contentment.DataEditors.
                 }
             }
 
+            // EDGE-CASE: To support the Social Links network configuration with the Cards display mode.
+            if (vm.displayMode === "cards") {
+                if (propertyName === "cardStyle" && item.value.hasOwnProperty("backgroundColor") === true) {
+                    return { "background-color": item.value.backgroundColor };
+                }
+
+                if (propertyName === "iconStyle" && item.value.hasOwnProperty("iconColor") === true) {
+                    return { "color": item.value.iconColor };
+                }
+            }
+
```

Comment:
I'm not entirely happy about having edge-case logic in the Configuration Editor controller.
I'd like to explore a cleaner approach, but I didn't want this to become a showstopper for the Social Links feature.

Reply:
Resolved with commit 2c43bf1724bca76e283315065936e229a84cfd0e - we can set additional AngularJS expression templates at `ConfigurationEditorModel` level.